README.md

```diff
@@ -1,24 +1,26 @@
-# GitHub Actions for Fermyon/Spin: fermyon/actions
+# `fermyon/actions` - GitHub Action collection for Spin and Fermyon Cloud
 
-The `fermyon/actions` is a collection of Github actions to enable running [spin](https://github.com/fermyon/spin) commands in your GitHub Actions workflow.
+With the `fermyon/actions` collection, you can incorporate [Spin](https://developer.fermyon.com/spin/index) and [Fermyon Cloud](https://developer.fermyon.com/cloud/index) in your [GitHub Action](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow). 
 
-It enables the following usecases:
+This collection of Actions enables the following use cases:
 
-- setup spin and plugins (`fermyon/actions/spin/setup`)
-- build and push to OCI registry (`fermyon/actions/spin/push`)
-- deploy to Fermyon Cloud (`fermyon/actions/spin/deploy`)
+- [x] setup Spin CLI and plugins using [`fermyon/actions/spin/setup`](#install-spin-cli-and-plugins---fermyonactionspinsetup)
+- [x] build and push your Spin app to an OCI registry using [`fermyon/actions/spin/push`](#push-spin-app-to-an-oci-registry---fermyonactionspinpush)
+- [x] deploy your Spin app to Fermyon Cloud using [`fermyon/actions/spin/deploy`](#deploy-spin-app-to-fermyon-cloud---fermyonactionspindeploy)
 
-## `fermyon/action/spin/setup`
+Let's take a look at each one to learn about the required inputs and walk through an example. 
+
+## Install Spin CLI and Plugins - `fermyon/action/spin/setup`
```

Comment:
On line 7 it says `fermyon/actions...` plural rather than `fermyon/action...` singular.  Which is correct?

(This affects the other subheadings too, and this may affect the internal links on lines 7-9.)